src/base/bittorrent/session.cpp

```diff
@@ -3929,6 +3929,33 @@ quint64 Session::getAlltimeUL() const
     return m_statistics->getAlltimeUL();
 }
 
+Session::OrganizedStats Session::getOrganizedStats() const
```

Comment:
I strongly object to having both "organized" and "separated" statistics getters in Session interface.
It is necessary to decide which of the forms is more suitable, and keep it only.
In case if we prefer to keep "separated" form we can add some additional getters for convenience (e.g. globalRatio, readCacheHits, etc., which are calculated currently) and change the units of some other fields (assuming we don't use existing "low-level" units somewhere else).